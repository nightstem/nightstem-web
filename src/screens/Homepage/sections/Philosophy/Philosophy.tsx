import cn from '@/utils/classnames';

const Philosophy = () => {
  return (
    <div className="grid screen-section items-center justify-end overflow-hidden px-2 lg:pr-24">
      <section className="relative screen-container">
        <div
          className={cn(
            'absolute top-1/2 left-1/2 -z-10 -translate-1/2',
            'aspect-square max-square-size rounded-full',
            'bg-radial from-secondary-500/15 from-0% to-secondary-500/0 to-100% blur-2xl',
          )}
        />

        <div className="flex measure-body flex-col gap-7">
          <h2 className="heading-lg md:text-right md:heading-xl">
            Built After Hours
          </h2>

          <p className="w-full max-w-[48ch] md:text-right">
            Nightstem is a studio for focused software. We value clarity and
            intent over noise and distraction. Built after hours, our projects
            are shaped with care and designed to last.
          </p>
        </div>
      </section>
    </div>
  );
};

export default Philosophy;
