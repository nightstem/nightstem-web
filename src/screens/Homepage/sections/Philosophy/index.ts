export { default as Philosophy } from '@/screens/Homepage/sections/Philosophy/Philosophy';
