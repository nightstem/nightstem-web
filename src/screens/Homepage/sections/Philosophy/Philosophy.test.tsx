import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';

import Philosophy from '@/screens/Homepage/sections/Philosophy/Philosophy';

describe(Philosophy, () => {
  it('matches the snapshot', () => {
    const { container } = render(<Philosophy />);

    expect(container).toMatchSnapshot();
  });
});
