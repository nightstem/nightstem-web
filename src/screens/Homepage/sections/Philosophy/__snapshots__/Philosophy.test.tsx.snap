// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Philosophy > matches the snapshot 1`] = `
<div>
  <div
    class="grid screen-section items-center justify-end overflow-hidden px-2 lg:pr-24"
  >
    <section
      class="relative screen-container"
    >
      <div
        class="absolute top-1/2 left-1/2 -z-10 -translate-1/2 aspect-square max-square-size rounded-full bg-radial from-secondary-500/15 from-0% to-secondary-500/0 to-100% blur-2xl"
      />
      <div
        class="flex measure-body flex-col gap-7"
      >
        <h2
          class="heading-lg md:text-right md:heading-xl"
        >
          Built After Hours
        </h2>
        <p
          class="w-full max-w-[48ch] md:text-right"
        >
          Nightstem is a studio for focused software. We value clarity and intent over noise and distraction. Built after hours, our projects are shaped with care and designed to last.
        </p>
      </div>
    </section>
  </div>
</div>
`;
